Apps/frontend/src/labels.js

```diff
@@ -8,6 +8,13 @@ export const LABEL_BUTTON_LOAD_CHANNEL_CONFIG = "Load";
 export const LABEL_BUTTON_UPDATE_CHANNEL_CONFIG = "Save";
 export const LABEL_BUTTON_CANCEL_CHANNEL_CONFIG = "Cancel";
 
+export const TOOLTIP_BUTTON_GND = "GND";
+export const TOOLTIP_BUTTON_DISTRIBUTE = "Distribute";
+export const TOOLTIP_BUTTON_RESET = "Reset";
```

Comment:
Maybe a little more descriptive labels. Especially the ones that should contain a bit more information. What is distributed? What is reset?

Reply:
Isn't reset self-explanatory, what text would you prefer elsewise?

Reply:
I don't think so. After all, it doesn't reset everything like the sliders, for example. Maybe you should write "Cleans the canvas and stops plotting"?

Reply:
Okay, sounds good. I'll change the label text :)